sanket/javascript/assignment12(day25)/sanketA12_Q2.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>employee details constructor</title>
+  </head>
+  <body>
+    <script>
+      class Employee {
+        constructor(empNo, name, address, city, salary) {
+          this.empNo = empNo;
+          this.name = name;
+          this.address = address;
+          this.city = city;
+          this.salary = salary;
```

Comment:
Same here also.

Reply:
same here

Reply:
Ok

Reply:
need to create an empty constructor here, as per question